### ISSUE TITLE
Strange behavior when just installed windows using Microwin iso

### ISSUE THREAD

**Issue by quan1232004** [ChrisTitusTech/winutil#2334]
**Describe the bug**
When the oobe is done I can only see black screen with file explorer. I had to restart twice to get to normal windows. The start menu there is still all of the shortcuts for promotional apps and on the installed apps I still see Microsoft Edge lying in there despite it being removed

**To Reproduce**
Just create Microwin unticking all options and import drivers like in #2333 

**Expected behavior**

Boot normally 

**Screenshots**

**Additional context**

**Comment by gpstarman** [comment]
This also happened to me repeatedly for two installs (that's all I did). I presume that it was doing some post install things, so I just waited for few minutes. Then the background appeared, but still nothing works. So, I restarted. Now everything works fine. I imported the drivers too (#2236). But IDK if it's related to drivers.







https://github.com/user-attachments/assets/8cfcf407-16d7-481d-8a4d-bfcfe87dbf39




bad video quality = github doesn't allow vids over 10mb and HEVC vids doesn't even play.
